ajax/libs/radium/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "radium",
+  "version": "0.17.1",
+  "description": "A set of tools to manage inline styles on React elements",
+  "filename": "radium.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/formidablelabs/radium.git"
+  },
+  "homepage": "http://stack.formidable.com/radium/",
+  "dependencies": {
+    "array-find": "^1.0.0",
+    "exenv": "^1.2.0",
+    "inline-style-prefixer": "^1.0.3",
+    "rimraf": "^2.4.0"
+  },
+  "keywords": [
+    "react",
+    "style",
+    "inline"
+  ],
+  "license": "MIT",
+  "authors": [
+    "Alex Lande",
+    "Ian Obermiller"
+  ],
+  "npmName": "radium",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|map)"
```

Comment:
there was `.gzip` in dist in an old case, i think both `**/*` and `**/*.+(js|map)` are fine.